core_contracts/loans/loans/loans.py

```diff
@@ -449,6 +466,24 @@ def getBnusdValue(self, _name: str) -> int:
         loop_value = self._positions._snapshot_db[-2].total_mining_debt.get()
         return EXA * loop_value // bnUSD_price
 
+    @external
+    @only_governance
+    def setMaxRetireAmount(self, _sicx_value: int, _bnusd_value: int) -> None:
+        """
+        :param _sicx_value: Maximum sICX amount to retire.
+        :param _bnusd_value: Maximum bnUSD amount to retire.
+        Sets the Maximum sICX amount and Maximum bnUSD amount to retire.
+        """
+        self._max_sicx_retire.set(_sicx_value)
+        self._max_bnusd_retire.set(_bnusd_value)
+
+    @external(readonly=True)
+    def getMaxTokensRetireAmount(self) -> dict:
```

Comment:
Please remove 'Tokens' from the method name. It just confuses since you have to ask why it is different from the setter. and other questions.